packages/moonstone/styles/variables.less

```diff
@@ -176,6 +184,13 @@
 @moon-divider-border-width: 3px;
 @moon-divider-height: 39px;
 
+// DropDown
+// ---------------------------------------
+@moon-dropdown-border-radius: 3px;
```

Comment:
Let's base this value on the Button border-radius of `@moon-button-border-radius`, so their relationship is established from the start.
And let's also do that with the next line: `@moon-dropdown-width: @moon-button-max-width`